### DISCUSSION TITLE
How can I run some scripts when dialogue finished?

### DISCUSSION THREAD

**Discussion by GNUqb114514** [nathanhoad/godot_dialogue_manager#186] [Q&A] [answered]
I am making a game, I need to run some scripts when dialogue finished.
But I have not any methods to finish it.
How can I run some scripts when dialogue finished?

**Comment by GNUqb114514** [comment]
P.S. I'm using DialogueManager 1.7.6 in Godot 3.5.2.

**Reply by GNUqb114514** [reply]
sorry, 1.17.6.

**Comment by nathanhoad** [comment, marked as answer]
In Dialogue Manager 1.x you can use the `dialogue_finished` signal to know when dialogue has ended: https://github.com/nathanhoad/godot_dialogue_manager/blob/v1.x/docs/Using_Dialogue.md#signals - Just note that if you are running multiple dialogues at once you might encounter the signal earlier than expected.

**Reply by GNUqb114514** [reply]
Thank you.
